### PR TITLE
Ticket #82 part ii + test cases fix based on inputvalidationcontroller update

### PR DESCRIPTION
**Pull request by jerdbaer**

### CODE REVIEW

File: database/AddWorkingTime.java [hunk 1]
```diff
@@ -83,18 +84,18 @@ private void createConnection() {
...
-	 * @param beginTime  ist der Beginn der Arbeitszeit im Format hh:mm:ss
-	 * @param endTime    ist das Ende der Arbeitszeit im Format hh:mm:ss
-	 * @param totalBreak ist die Gesamtzeit der Pausen an dem Tag im Format hh:mm:ss
-	 * @param overtime   ist die Anzahl der Überstunden, die an dem Tag geleistet
-	 *                   wurden im Format hh:mm:ss
-	 * @param comment    ist der Kommentar zum Arbeitszeiteintrag
-	 * @throws BatchUpdateException wenn bereits ein Eintrag für den MA und das
-	 *                              Datum existiert (wird extern abgefangen)
-	 * @return Rückmeldung, ob der Eintrag erfolgreich angelegt wurde
+	 * @param beginTime  is the begin of working time at a working days in hh:mm:ss
```
Comment (valientes) [inline database/AddWorkingTime.java]: singular day (also in line below at endTime)

File: database/CheckPassword.java [hunk 2]
```diff
@@ -125,14 +125,17 @@ public static boolean addAccount(int MA_ID, String pw){
...
+   * Overwrites the stored password for an existing employee's account in case the old password was provided correctly 
+   * by the user's input
    *
-   * @param MA_ID ist der Account, für den das Passwort geändert werden soll
-   * @param oldPw ist das alte Passwort
-   * @param newPw ist das neue Passwort
-   * @return Angabe, ob das Passwort erfoglreich geändert wurde
-   * @throws FalsePasswordException wenn {@code oldPw} falsch ist
-   * @throws NoChangeException wenn {@code oldPw == newPw}
+   * @param MA_ID employee' account for which the password should be overwritten
```
Comment (valientes) [inline database/CheckPassword.java]: s is missing at employee's (or ' too much)

File: database/GetOvertime.java [hunk 3]
```diff
@@ -108,6 +110,15 @@ public String getSum(int MA_ID){
...
    }
 
+   /**
+    * Selects the time of working end of the database record on the day before the selected date for the new working 
+    * time record for user input validation
+    * 
+    * @param MA_ID individual employee's id in int
+    * @param yesterday date of the day before the selected date for working time record
+    * 
+    * @return time of working end of the previous day as String String in hh:mm:ss
```
Comment (valientes) [inline database/GetOvertime.java]: If there is no entry for the previous day, `00:00:00` will be returned to ensure that there will be no error in validation. That should be added to the documentation

File: database/GetOvertime.java [hunk 4]
```diff
@@ -133,6 +144,15 @@ public String getWorkEndYesterday(int MA_ID, String yesterday){
...
    }
 
+   /**
+    * Selects the time of working begin of the database record on the day after the selected date for the new working 
+    * time record for user input validation
+    * 
+    * @param MA_ID individual employee's id number in int
+    * @param tomorrow date of the day after the selected date for working time record
+    * 
+    * @return time of working begin of the following day as String String in hh:mm:ss
```
Comment (valientes) [inline database/GetOvertime.java]: If there is no entry for the following day, `23:59:59` will be returned to ensure that there will be mistake in validation

**Review by valientes — CHANGES_REQUESTED** [review]
2 rather unimportant little spelling/grammar issues, 2 rather important additions to documentation

**Review by TomxHWRxWI20C — APPROVED** [review]
lgtm

File: database/AddWorkingTime.java (hunk 1 above)
Comment (jerdbaer) [reply database/AddWorkingTime.java]: fixed type

**Review by valientes — APPROVED** [review]

**Pull request merged**